### DISCUSSION TITLE
how to make shadow in perspective for rotating object

### DISCUSSION THREAD

**Discussion by pragavame** [google/model-viewer#3469] [Ideas]
Hey! how to make shadow in perspective for rotating object?

![shadows1](https://user-images.githubusercontent.com/99244643/168518269-d2491fe6-1304-4561-891b-8c2264b857f1.png)

**Comment by elalish** [comment]
We don't support this currently - it would hurt performance and the API to control it would need a lot of surface area.

**Reply by pragavame** [reply]
maybe do it as an additional API?

**Reply by elalish** [reply]
PRs welcome.
